invenio.cfg

```
@@ -202,8 +203,8 @@ CELERY_BEAT_SCHEDULE = {
         },
         "schedule": METRICS_CACHE_UPDATE_INTERVAL,
     },
-    "sitemap-updater": {
-        "task": "zenodo_rdm.sitemap.tasks.update_sitemap_cache",
+    "update_sitemap": {
```

all other task keys use "-"
```suggestion
    "update-sitemap": {
```